### ISSUE TITLE
Bug: The "Extrude" checkbox is redundant

### ISSUE THREAD

**Issue by raindropsfromsky** [jayanam/fast-carve#49]
The **Primitive** section of the addon has two check boxes: Extrude and fill.
Out of these, the "fill" check box is justified (there is a difference between ticked and clear).
But what does the "Extrude" check box achieve?

If the **Extrude** check box is clear, then pressing E extrudes the primitive, and creates a frame. But when we press **CTRL+LMB click**, it does not generate a shape. Although this outcome is logical, what is the actual use of this?

In other words, we have the existing geometry, and a frame (a 3D extrusion of the primitive).
We can grab and move this frame.  But beyond that, the addon does not let us use this frame in creating new geometry.

Therefore, either this **Extrude** check box should be removed, or we should be able to use the empty extruded frame for subsequent operations.
